partiql-tests-data/eval/primitives/operators/like.ion

```diff
@@ -1569,6 +1569,32 @@ like::[
       ]
     }
   },
+  {
+    name:"like expression value, pattern, and escape as variable references",
+    statement:'''
+      SELECT v FROM
```

Comment:
since `v` is the same in two test cases, maybe change `v` in one of them to make it explicit which row is being filtered, or do something like below:

statement:
```
SELECT v LIKE p ESCAPE e as is_like FROM [
   {'v': 'abc%', 'p': 'abc/%', 'e': '/'},
   {'v': 'abcd', 'p': 'abc%', 'e': '/'},
   {'v': 'abcd', 'p': 'abc', 'e': '/'}
] AS tbl
   ```

result:
```
[
  {
    'is_like': true
  },
  {
    'is_like': true
  },
  {
    'is_like': false
  }
]
```

Reply:
Yeah, making it explicit will make the test clearer. I'll use your example statement.